Modules/Findliblcf.cmake

```diff
@@ -0,0 +1,34 @@
+find_path(LIBLCF_INCLUDE_DIR_INTERNAL liblcf/reader_lcf.h)
+find_library(LIBLCF_LIBRARY NAMES lcf liblcf)
+if(EXISTS "${LIBLCF_INCLUDE_DIR_INTERNAL}")
+	set(LIBLCF_INCLUDE_DIR "${LIBLCF_INCLUDE_DIR_INTERNAL}/liblcf")
+endif()
+
```

Comment:
Not really sure this does the right thing...

Reply:
I will remove that liblcf module and will directly write a FindModule for the liblcf repo. (since xyzimage I know how to write one \*g\*)